src/scss/components/optional/_cookie-warning.scss

```diff
@@ -0,0 +1,49 @@
+@mixin cookieWarning() {
+	.c-cookieWarning {
+	    background-color: $black;
+	    position: fixed;
+	    bottom: 0;
+		width: 100%;
+		$zIndex: zIndex(high);
+	}
+
+	.c-cookieWarning-inner {
+	    width: 940px;
+	    margin: 0 auto;
+	    padding: spacing() 0;
+	    overflow: hidden;
+
+    	@include media('<960px') {
+	        width: 620px;
+		}
+		@include media('<640px') {
+	        width: 300px;
+		}
+	}
+
+	.c-cookieWarning p {
+		color: $white;
+		font-size: $font-size-base;
```

Comment:
Is this needed? should inherit the base size from above ☁️

Reply:
👍